docs/private-ddn/ddn-workspace.mdx

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 5
+sidebar_label: DDN workspace
+description: "Learn how to develop with DDN workspace"
+keywords:
+  - hasura ddn
+  - private ddn
+  - dedicated vpc
+  - enterprise ddn
+  - create private ddn
+  - ddn workspace
+seoFrontMatterUpdated: false
+---
+
+import Thumbnail from "@site/src/components/Thumbnail";
+
+# Developing with DDN workspace
+
+## Introduction
+
+DDN workspace is a development environment to work with on Private DDN on the browser, without having to install DDN CLI and its dependencies in your local machine. It has a familiar VS Code like UI, runs on your data plane and has all the dependencies required to work with DDN pre-installed. 
+
+This guide will walk you through the steps for using DDN workspace to develop your API on a Data Plane in
+[Private Hasura DDN](/private-ddn/overview.mdx). Before proceeding, you'll first need to be
+[invited](/private-ddn/data-plane-collaboration.mdx) to a Data Plane.
+
+:::info Prerequisite
+
+To provision workspaces in Private DDN, please reach out to us 
+[here](https://hasura.io/contact-us).
+
+:::
+
+The recommendation is to have a workspace dedicated per developer working on Private DDN API development. Once the workspace has been provisioned, it can be launched from the Private DDN page.
+
+## Advantages
+
+- Get started with just your browser. No need to install DDN CLI and its dependencies (Docker) in your local machine.
+- Network connectivity - As the workspace is running on your data plane, it has access to all the data sources and services available in the data plane network. No need to setup complex VPN/network policies to access your DBs from your local machine.
+- Collaboration - Multiple developers can work on the same project simultaneously in the same workspace, if needed.
+
+
+## Step 1. Launch a workspace
+
+To launch a workspace, click on the `Launch Workspace` button on the Private DDN page. This will open a new tab in your browser with the workspace UI. Click on the `Copy Password` button to copy the password to your clipboard. Use this password to login to the workspace.
```

Comment:
```suggestion
- To launch a workspace, click on the `Launch Workspace` button on the Private DDN. The list of all the Private DDN accessible to you are available [here](https://console.hasura.io/data-plane). This will open a new tab in your browser with the workspace UI.

- Click on the `Copy Password` button to copy the password to your clipboard. Use this password to login to the workspace.
```